strapi/strapi
# [CustomField] Add custom fields

### What does it do?

Adds custom fields to the Strapi admin as described in [RFC #42](https://github.com/strapi/rfcs/blob/main/rfcs/0042-custom-fields-api.md)

### How to test it?

- Go to any content type
- Click create an attribute, select the custom tab, and choose the Color custom field
- Save the content type
- Head over to the content manager and create an entity
- You should see a color picker input for your color custom field
- You should be able to select a color and see the change in the input
- You should be able to save the entity and see the value in the db / api response

### Related PRs
- https://github.com/strapi/strapi/pull/14443 must be merged before the stable release
